src/Esprima/Utils/AstVisitor.cs

```diff
@@ -1,15 +1,34 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
 using Esprima.Ast;
-using static Esprima.EsprimaExceptionHelper;
 
 namespace Esprima.Utils
 {
     public class AstVisitor
     {
-        public bool IsStrictMode { get; set; } = false;
+        private readonly List<Node> _parentStack = new List<Node>();
```

Comment:
Doesn't seem to be used. Am I missing something?

Reply:
This is used to track parent nodes stack instead of `NodeExtensions.AncestorNodes` call, but I think it is not necessary here as could be implemented easily by deriving from visitor. Should I remove it?

Reply:
Maybe if it's unused, private and not referenced might be easier to remove 🙂

Reply:
done